crates/core_simd/examples/README.md

```diff
@@ -0,0 +1,109 @@
+### `stdsimd` examples
+
+This crate is a port of example uses of `stdsimd`, mostly taken from the `packed_simd` crate.
+
+The examples contain, as in the case of `dot_product.rs`, multiple ways of solving the problem, in order to show idiomatic uses of SIMD and iteration of performance designs.
+
+Run the tests with the command 
+
+```
+cargo run --example dot_product
+```
+
+and verify the code for `dot_product.rs` on your machine.
+
+### `dot_product.rs`
+
+This example code takes the dot product of two vectors. You are supposed to mulitply each pair of elements and add them all together.
+
+The easiest way to inspect the assembly of the `scalar` code versions (the non-SIMD versions) is to [click this link](https://rust.godbolt.org/z/xM9Mxb14n) for a *mise en place* of what is going on.
```

Comment:
I think it would be better to avoid non-english phrases since not everyone knows French (i guess? idk what that phrase means).